.github/workflows/tests_build.yml

```diff
@@ -85,7 +84,6 @@ jobs:
           cd python
           make -j8
           python3 ../../doc/test-suite.py ${{ github.workspace }}/${{env.GH_PAGES_DIR}}/_tests ${{ github.workspace }}/${{ env.TEMP_DIR }}/${{ env.DEV_DIR }}/
-          ../../doc/test-suite-to-png.sh ${{ github.workspace }}/${{ env.TEMP_DIR }}/${{ env.DEV_DIR }}/
```

Comment:
This is the problem. If you put it back it should work. It can then be removed in a subsequent PR.

Reply:
Of course!